src/main/java/ir/sahab/hbaserule/HbaseRule.java

```diff
@@ -44,25 +46,21 @@ private HbaseRule() {
 
     @Override
     protected void before() throws Throwable {
-        utility = new HBaseTestingUtility();
-        Configuration configuration = utility.getConfiguration();
-        configuration.set(HConstants.HBASE_CLIENT_RETRIES_NUMBER, "1");
-        if (customConfigs != null) {
-            customConfigs.forEach(configuration::set);
-        }
+        customConfiguration.set(HConstants.HBASE_CLIENT_RETRIES_NUMBER, "1");
```

Comment:
Put all changes in configuration in a single block of code with this section block:
// Set necessary HBase configuration items